src/test/java/roomescape/ReservationsTest.java

```diff
@@ -0,0 +1,58 @@
+package roomescape;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import roomescape.model.Reservation;
+import roomescape.model.ReservationDateTime;
+import roomescape.model.Reservations;
+
+class ReservationsTest {
+
+    @DisplayName("예약을 추가할 수 있다.")
+    @Test
+    void can_add_reservation() {
+        Reservations reservations = new Reservations();
+
+        reservations.add(
+                new Reservation(1L, "포스티", new ReservationDateTime(
+                        LocalDateTime.of(
+                                LocalDate.of(2025, 4, 16),
+                                LocalTime.of(11, 0)
+                        )))
```

Comment:
LocalDate와 LocalTime 을 의도적으로 분리하신 걸까요?
아래처럼 줄여서 표현할 수도 있습니다~

```suggestion
                        LocalDateTime.of(2025, 4, 16, 11, 0)))
```